UI/Web/src/app/user-settings/api-key/api-key.component.html

```diff
@@ -4,7 +4,7 @@
     <div class="input-group">
         <input #apiKey type="text" readonly class="form-control" id="api-key--{{title}}" aria-describedby="button-addon4" [value]="key" (click)="selectAll()">
         <div class="input-group-append" id="button-addon4">
-            <button class="btn btn-outline-secondary" type="button" (click)="copy()"><span class="sr-only">Copy</span><i class="fa fa-copy" aria-hidden="true"></i></button>
+            <button class="btn btn-outline-secondary" type="button" (click)=copy()><span class="sr-only">Copy</span><i class="fa fa-copy" aria-hidden="true"></i></button>
```

Comment:
Looks like you're missing the "" around the copy call. Please add them in.

Reply:
Resolved in 8922be2